dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/ShutdownServiceImpl.java

```diff
@@ -140,7 +140,7 @@ public Object call() {
                 final CountDownLatch latch = new CountDownLatch(1);
                 try {
 
-                    String jobId = "";
+                    final String jobId = "";
```

Comment:
I think there is a different problem here.  At one point this probably was overwritten, but after some refactoring it wasn't reassigned.  So if you look at line 162, the empty jobId is passed in.  We should make sure that there isn't a bug here with that.